src/main/java/net/proselyte/springbootdemo/Controller/BookController.java

```diff
@@ -0,0 +1,91 @@
+package net.proselyte.springbootdemo.Controller;
+
+import javax.validation.Valid;
+import lombok.AllArgsConstructor;
+import net.proselyte.springbootdemo.Model.Book;
+import net.proselyte.springbootdemo.Service.BookService;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.domain.Sort.Direction;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+@AllArgsConstructor
+public class BookController {
+  private final BookService bookService;
+  private final PageableHelper<Book> pageableHelper;
+
+  @GetMapping("/books")
+  public String getBooks(Model model,
+      @RequestParam(value = "pageNumber", required = false, defaultValue = "0") int pageNumber,
+      @RequestParam(value = "sortField", required = false, defaultValue = "title") String sortField,
+      @RequestParam(value = "sortType", required = false) String sortType) {
+
+    Direction sortDirection = sortType == null ? Direction.ASC : Direction.fromString(sortType);
+    PageRequest pageRequest = PageRequest.of(pageNumber, 10, Sort.by(sortDirection, sortField));
+    Page<Book> page = bookService.getBooksPage(pageRequest);
+    pageableHelper.fillPageable(model, page, sortField, sortDirection);
+    return "books.html";
+  }
+
+  @GetMapping("/book/{bookId}")
+  public String deleteBook(@PathVariable("bookId") long bookId) {
+    bookService.deleteBook(bookId);
+    return "redirect:/books";
+  }
+
+  @GetMapping("/book/{userId}/{bookId}")
+  public String getBookBack(@PathVariable("bookId") long bookId, @PathVariable("userId") long userId) {
+    bookService.getBookBack(bookId, userId);
+    return "redirect:/user/" + userId + "/books";
+  }
+
+  @PostMapping("/book/issue")
+  public String issueBook(long bookId, long userId) {
```

Comment:
move to user and change to path variable

`@PostMapping("/user/{userId}/issue-book/{bookId}"`